### PR TITLE
AQ-184 BigListOfNaughtyStrings test

### PR DESCRIPTION
**Pull request by byronlin-bq**

### CODE REVIEW

File: IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+                    throw e;
+                }
+            }
+
+            if (tv.ValidIon == null || tv.ValidIon.Value == true)
+            {
+                TestUtil.AssertEquals(
+                    HashWriter.Digest(),
+                    HashReader.Digest(),
+                    "Reader/writer hashs for line |" + tv.AsIon() + "| as |" + s + "| don't match");
```
Comment (simonz-bq) [inline IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs]: Typo?

File: IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+            }
+        }
+
+
+        public class CustomDataSourceAttribute : Attribute, ITestDataSource
```
Comment (simonz-bq) [inline IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs]: I think the name should be more descriptive.

File: IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs (hunk 1 above)
Comment (byronlin-bq) [reply IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs]: Addressed

File: IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs (hunk 2 above)
Comment (byronlin-bq) [reply IonHashDotnet.Tests/BigListOfNaughtyStringsTest.cs]: Addressed

**Pull request merged**